nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py

```diff
@@ -159,7 +169,7 @@ def _metrics_require_string2category_map(self):
     def setup(self, stage=None):
         # NOTE: super().__init__ will try and setup train/val/test datasets, but we sidestep this using a if self._train_ds is not None condition
         # We then set things up for real only once setup() of this class is called.
-        resume_checkpoint_path = self.trainer._checkpoint_connector.resume_from_checkpoint_fit_path
+        resume_checkpoint_path = self.trainer.ckpt_path
```

Comment:
hi @stevehuang52 do you  remember whether and why this change is necessary ?

Reply:
I think we need to first merge main into our branch, resolve the conflicts, then merge our branch into main, since the main branch has changed a lot since we last pull

Reply:
@zhehuaichen just noticed that you're merging into your own main branch, not the latest NeMo main. In this case, you can just keep all changes in the `Megatron_gpt_sft_model.py` for now. The original PR where I copied the changes from is this one [6915](https://github.com/NVIDIA/NeMo/pull/6915), but I made a few changes to it to keep it work with PTL 1.x instead of 2.x used in NeMo main.